templates/phaser/packages/client/package.json

```diff
@@ -38,6 +38,7 @@
     "@types/react-dom": "18.2.7",
     "@types/styled-components": "^5.1.26",
     "@vitejs/plugin-react": "^3.1.0",
+    "eslint": "8.57.0",
```

Comment:
What was the issue with `eslint`?

Reply:
That it wouldn't let me commit because it had some kind of problem. I can try to reproduce it, but it'll take a bit.

Reply:
if you can't repro, should we remove this?